evalai/utils/common.py

```diff
@@ -66,4 +66,5 @@ def clean_data(data):
     """
     data = BeautifulSoup(data, "lxml").text.strip()
     data = ' '.join(data.split())
```

Comment:
```
data = ' '.join(data.split()).encode("utf-8")
return data
```

I just realised that we can reduce one statement. @inishchith Can you please update it.